plugins/modules/ecs_service.py

```diff
@@ -697,7 +724,18 @@ def main():
         launch_type=dict(required=False, choices=['EC2', 'FARGATE']),
         platform_version=dict(required=False, type='str'),
         service_registries=dict(required=False, type='list', default=[], elements='dict'),
-        scheduling_strategy=dict(required=False, choices=['DAEMON', 'REPLICA'])
+        scheduling_strategy=dict(required=False, choices=['DAEMON', 'REPLICA']),
+        propagate_tags=dict(required=False, choices=['TASK_DEFINITION', 'SERVICE']),
+        tags=dict(
+            required=False,
+            default=[],
+            type='list',
+            elements='dict',
+            options=dict(
+                key=dict(type='str', no_log=True),
+                value=dict(type='str'),
+            )
+        )
```

Comment:
Please use a standard dict here.  There's a helper "ansible_dict_to_boto3_tag_list" which will do the conversion for you.